### PR TITLE
Optimizely Data Platform - New docs

### PR DESCRIPTION
**Pull request by Romani-Ramzi**

### CODE REVIEW

File: src/connections/destinations/catalog/actions-optimizely-data-platform/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+2. (Optional) Enter a descriptive name for the event.
+3. In section 1, select the event that you want to send to ODP.
+4. In section 2, click **Load Test Event from Source**. This generates the raw data for the selected event and populates the mappings in section 3.
+5. In section 3, the ID and timestamp field mappings auto-populate, but you can edit them as desired. Ensure you select the event type and, optionally, the event action. For example, if you are configuring a custom event to track button clicks, select _button_ for the event type, and _click_ for the event action.
+   > 📘 Note
+   > 
+   > In ODP, each event requires an ID, timestamp, and event type. The event action is optional. See [Events](https://docs.developers.optimizely.com/optimizely-data-platform/docs/thebasics-events) in the ODP documentation.
+6. (Optional) To ensure the custom event is configured correctly, in section 4, click **Send test event to destination**.
+7. Click Save.
+8. Toggle the **Status** to **Enabled** for the custom event you created.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-optimizely-data-platform/index.md:48]: The note messes with the markdown lists - this should fix the numbering issue. 

```suggestion
In Twilio Segment, on the **Mappings** tab of the Optimizely Data Platform destination, Segment displays a list of pre-built mappings that you can enable or disable. For example, if you enabled the **Email Opened** mapping, each email opened event Segment ingested after you enabled the mapping would sync to ODP.

If you want to map an event that is not listed:
1. Click **New Mapping > Custom Event**.
2. _(Optional)_: Enter a descriptive name for the event.
3. Select the event that you want to send to ODP.
4. Click **Load Test Event from Source**. This generates the raw data for the selected event and populates your mappings. The ID and timestamp field mappings auto-populate, but you can edit them as desired.
5. Select the event type and, optionally, the event action. For example, if you are configuring a custom event to track button clicks, select _button_ for the event type and _click_ for the event action.

> info "Required fields"
> In ODP, each event requires an ID, timestamp, and event type. The event action is optional. See ODP's [Events](https://docs.developers.optimizely.com/optimizely-data-platform/docs/thebasics-events) documentation for more details.

<ol style="counter-reset: none;">
  <li value="6" markdown=1>
  _(Optional)_: To ensure the custom event is configured correctly, click **Send test event to destination**.
  </li>
  <li value="7" markdown=1>
  Click **Save**.
  </li>
  <li value="8" markdown=1>
  Toggle your custom event's status to **Enabled**.
  </li>
</ol>	
```

**Review by forstisabella — CHANGES_REQUESTED** [review]

**Review by forstisabella — APPROVED** [review]

File: src/connections/destinations/catalog/actions-optimizely-data-platform/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  <li value="7" markdown=1>
+  Click **Save**.
+  </li>
+  <li value="8" markdown=1>
+  Toggle your custom event's status to **Enabled**.
+  </li>
+</ol>	
+
+The event data sends from Twilio Segment to ODP starting after you enable the mapping in the destination. It does not retroactively send events that occurred prior to configuring the integration and enabling the mappings.
+
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-optimizely-data-platform/index.md:62]:
```suggestion

{% include components/actions-fields.html %}
```

This snippet will populate the available action on the docs page^

File: src/connections/destinations/catalog/actions-optimizely-data-platform/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---
+title: Optimizely Data Platform Destination
+beta: true
+hidden: true
+id: 6512d7f86bdccc3829fc4ac3
+---
+
+Sync your Twilio Segment customer data to Optimizely Data Platform (ODP) for real-time segmentation, reporting, and to enrich customer profiles in ODP.
+
+After you set up your Optimizely Data Platform destination, Segment syncs your customer data to ODP in near real-time.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/actions-optimizely-data-platform/index.md:10]:
```suggestion
After you set up your Optimizely Data Platform destination, Segment syncs your customer data to ODP in near real-time.

This destination is maintained by Optimizely. For any issues with the destination, [contact their Support team](mailto:support@<integration_name>.com).
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]
just two changes for me. lmk if you have any questions!

File: src/connections/destinations/catalog/actions-optimizely-data-platform/index.md (hunk 3 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/actions-optimizely-data-platform/index.md:10]: the mailto should be optimizely or segment?

Comment (forstisabella) [reply src/connections/destinations/catalog/actions-optimizely-data-platform/index.md:10]: It should be Optimizely, if you all are owning it!

**Comment by Romani-Ramzi** [comment]
> just two changes for me. lmk if you have any questions!

currently we would like to go with your suggestions, as you have more experience on what customers understand when reading through segment docs.

but our doc team will have to go through that as well, and raise what they think is good, hopefully they like your suggestions :).

File: src/connections/destinations/catalog/actions-optimizely-data-platform/index.md (hunk 3 above)
Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/actions-optimizely-data-platform/index.md:10]: got it, I will update now

Comment (tcgilbert) [reply src/connections/destinations/catalog/actions-optimizely-data-platform/index.md:10]: Customers are usually well versed enough to differentiate where to send their support queries. If something is wrong within the Segment UI they will almost always reach out to us. 

Additionally, our support team is always available to assist in customer queries if insight is needed from the Segment side of things. You can always reach out to partner-support@segment.com for help

Comment (Romani-Ramzi) [reply src/connections/destinations/catalog/actions-optimizely-data-platform/index.md:10]: perfect, we updated the required part, please check

**Comment by joe-ayoub-segment** [comment]
hi @forstisabella - Romani and Alex are keen to get this doc pushed live. We're going to move the Integration to Public Beta today. Does anything else need to be changed in this PR before it can go live?

**Comment by forstisabella** [comment]
no worries, @Romani-Ramzi! I'm going to go ahead and merge this. Thanks for your collaboration on this! :)

**Comment by Romani-Ramzi** [comment]
Thanks alot @forstisabella, its was a pleasure going through all this with you and the team. :D

**Pull request merged**